ASM/build.py

```diff
@@ -13,13 +13,14 @@
 
 parser = argparse.ArgumentParser()
 parser.add_argument('--pj64sym', help="Output path for Project64 debugging symbols")
-parser.add_argument('--compile-c', action='store_true', help="Recompile C modules")
-parser.add_argument('--dump-obj', action='store_true', help="Dumps extra object info for debugging purposes. Does nothing without --compile-c")
+parser.add_argument('--compile-c', action='store_true', help="Recompile C modules. This is the default")
```

Comment:
should this still be an argument given it's the default?

Reply:
I'd like to keep it in place for now to not break existing scripts or editor configs calling this.